nThElemLinkedList.py

```diff
@@ -1,48 +1,68 @@
 """
 Implement an algorithm to find the kth to last element of a singly linked list.
+"""
+class Node(object):
+    # Function to initialize the node object
+    def __init__(self, data=None, next_node=None):
+        self.data = data  # Assign data
+        self.next_node = next_node  # Initialize 
+                          # next as null
+    def get_data(self):
```

Comment:
In Python generally getters and setters are not used for public attributes. Anything not prefixed with an underscore is considered public. That is, `self.data` is public while `self._data` is private. This is not enforced by the runtime but code review and analysis tools (like pylint) would point these out as problematic.